intelmq/bots/experts/url2fqdn/expert.py

```diff
@@ -4,14 +4,24 @@
 
 # -*- coding: utf-8 -*-
 from urllib.parse import urlparse
+from typing import Optional, List
 
 from intelmq.lib.bot import ExpertBot
 
+DEPRECATION_WARNING = "This bot is deprecated and will be removed in version 4.0. Use 'URL Expert' bot instead."
```

Comment:
Please add this to the docs of the url2fqdn bot too. Then all users know that the URL expert is the preferred bot instead.

Reply:
Added to docs.